src/pretalx/submission/forms/submission.py

```diff
@@ -72,9 +72,14 @@ def _set_track(self, instance=None):
 
             access_code = self.access_code or getattr(instance, "access_code", None)
             if not access_code or not access_code.track:
-                self.fields["track"].queryset = self.event.tracks.filter(
+                filter = self.event.tracks.filter(
```

Comment:
Please rename this to `_filter` or `track_filter` – `filter` is a built-in method in Python, and even though we don't use it here in this method, we don't want to start overriding built-ins 😉